Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -122,6 +140,20 @@ private extension LayoutManager {
 
         return blockquoteRect
     }
+    
+    private func drawNestedBlockquote(in rect: CGRect, with context: CGContext, at depth: Int) {
+        
+        let quoteCount = blockquoteBorderColors.count
+        let index = depth < quoteCount ? depth : quoteCount-1
```

Comment:
I think this will be easier to write like this: ```min(depth, quoteCount-1)```

Reply:
oh yes much better 😅